src/presentation/http/router/note.test.ts

```diff
@@ -521,6 +542,139 @@ describe('Note API', () => {
     });
   });
 
+  describe('PATCH /note/:notePublicId/parent', () => {
+    test('Returns 200 when parent was successfully updated', async () => {
+      const childNotePublicId = '9OYDD9d4_Y';
+
+      const parentNotePublicId = 'Tocn1f7rQS';
+
+      const newParentNotePublicId = 'lMkesQ1gpg';
+
+      await global.db.truncateTables();
+
+      const creator = await global.db.insertUser({
+        email: 'a@a.com',
+        name: 'test user',
+      });
+
+      const childNote = await global.db.insertNote({
+        creatorId: creator.id,
+        publicId: childNotePublicId,
```

Comment:
Now it is create automatically, you don't need to create magic variable - remove it